go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/flanksource/flanksource-ui/apm-hub
 
 require (
+	github.com/elastic/go-elasticsearch/v8 v8.6.0
```

Comment:
I think we will need add in support for the https://github.com/opensearch-project/opensearch-go/ client as well, as go-elasticsearch will not talk to OpenSearch